deploy/marathon/resources/connected-car.json

```diff
@@ -40,18 +20,17 @@
       "disk": 10,
       "gpus": 0,
       "instances": 2,
-      "backoffSeconds": 1,
-      "backoffFactor": 1.15,
-      "maxLaunchDelaySeconds": 3600,
+      "upgradeStrategy": {
+        "maximumOverCapacity": 0,
+        "minimumHealthCapacity": 0.5
+      },
+      "killSelection": "YOUNGEST_FIRST",
       "env": {
-        "KAFKA_SERVICE_NAME": "broker.kafka.l4lb.thisdcos.directory:9092",
+        "RP_PLATFORM": "mesos",
+        "RP_NAMESPACE": "connected-car",
```

Comment:
This matches the folder/group your application is in. It lets the service locator know how to form addresses.